src/components/tasks/card/index.tsx

```diff
@@ -34,7 +34,16 @@ const Card: FC<Props> = ({
   const iconHeight = '25px';
   const iconWidth = '25px';
 
-  const cardClassNames = [classNames.card];
+function formattedDates(localEndsOn: Date){
+  const year = localEndsOn.getFullYear()
+  const month = localEndsOn.getMonth() + 1;
+  const day = localEndsOn.getDate();
+  const finalDate = [year, month, day].join('-');
+  return finalDate;
+}
+
+  const dates = !!localEndsOn ? formattedDates(localEndsOn) : '';
```

Comment:
name of the variable can be changed to strDate to give a better sense and the type can be defined as string.